src/frontend/src/App.tsx

```diff
@@ -17,13 +17,31 @@ function App() {
     isAuthenticated(),
   );
 
+  const mainContentRef = useRef<HTMLDivElement>(null); // Create a reference for the main content container
+  const skipToContentRef = useRef<HTMLButtonElement>(null);
+  const skipToContent = () => {
+    if (mainContentRef.current) {
+      mainContentRef.current.focus();
+    }
+  };
+
+  useEffect(() => {
+    if (skipToContentRef.current) {
+      skipToContentRef.current.focus(); // Set focus to the main content container when the route changes
+    }
+  }, [pathname]);
```

Comment:
I like the idea of pathname as dependency to check for url changes. However, this is bringing the button to focus everytime we come to the screen, which is not an expected behaviour for most of our users.
Can you explore a possibility of somehow reseting tabindex everytime the pathname changes?

Reply:
do a new update for this issue, could you please check again